GPT/gpt.talon

```diff
@@ -11,6 +11,12 @@ model <user.modelPrompt> [this] [{user.modelInsertionMethod}]$:
     result = user.gpt_apply_prompt(modelPrompt, text)
     user.gpt_insert_response(result, modelInsertionMethod or "")
 
+model blend:
```

Comment:
Not sure bout this spoken form. Would be tempted to use something like

- model blend clip
- model paste
- model paste blend

I don't feel strongly tho so if you've already aligned on this one then feel free to ship this spoken form

Reply:
I'm ok with `model blend clip`. I was starting to get used to the `model blend` so I might have a little bit of interference but I think that's fine.

Reply:
Updated it